### PR TITLE
WIP:Add auction comp to the menu for testing

### LINKED ISSUE #14: Make auction page

### PR DESCRIPTION
**Pull request by biggicode**

### CODE REVIEW

**Review by MariaTeo — APPROVED** [review]
Nice work!! 💯

**Comment by RadosavAntonio** [comment]
Looks good to me. Nice!

File: src/component/action-card/action-card.component.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const ActionCard = () => {
             </ACS.Price>
           </ACS.Price>
         </ACS.PriceContainer>
-        <ACS.CardButton>Submit a bit</ACS.CardButton>
+        <ACS.CardButton to="/auctionPage" >Submit a bit</ACS.CardButton>
```
Comment (marianzburlea) [inline src/component/action-card/action-card.component.jsx:25]: change to `/auction`
it is already a page

File: src/component/app/app.component.jsx [hunk 2]
```diff
@@ -24,6 +25,7 @@ const App = () => {
           <Home path="/" />
           <Auctions path="auctions" />
           <Sell path="sell" />
+          <AuctionPage path="auctionPage" />
```
Comment (marianzburlea) [inline src/component/app/app.component.jsx:28]: change path to `auction`

**Review by marianzburlea — APPROVED** [review]
next time commit your files separate from package.json and yarn changes

**Review by simonamarcu2 — APPROVED** [review]
great work 👍

**Pull request merged**